test/unit/uk/gov/hmrc/apidefinition/models/APIDefinitionSpec.scala

```diff
@@ -24,118 +24,104 @@ import uk.gov.hmrc.play.test.UnitSpec
 
       apiDefinition.versions.head.access shouldBe Some(PublicAPIAccess())
     }
 
     "read from JSON when the API access type is PUBLIC and there is an empty whitelist" in {
-      val access = """{
-          |  "type": "PUBLIC",
-          |  "whitelistedApplicationIds": []
-      }""".stripMargin
-      val apiDefinition = anApiDefinition(access = access)
+      val apiDefinition = anApiDefinition(whitelistedApplicationIds = Some("[]"))
 
       apiDefinition.versions.head.access shouldBe Some(PublicAPIAccess())
     }
 
     "fail to read from JSON when the API access type is PUBLIC and there is a non-empty whitelist" in {
-      val access = """{
-          | "type": "PUBLIC",
-          | "whitelistedApplicationIds": ["an-application-id"]
-      }""".stripMargin
-
-      intercept[RuntimeException] {  anApiDefinition(access = access) }
+      intercept[RuntimeException] {
+        anApiDefinition(whitelistedApplicationIds = Some("[\"an-application-id\"]"))
+      }
     }
 
     "read from JSON when the API access type is PRIVATE and there is an empty whitelist" in {
-      val access = """{
-          | "type": "PRIVATE",
-          | "whitelistedApplicationIds": []
-      }""".stripMargin
-
-      val apiDefinition = anApiDefinition(access = access)
+      val apiDefinition = anApiDefinition(
+        accessType = "PRIVATE", whitelistedApplicationIds = Some("[]"))
 
       apiDefinition.versions.head.access shouldBe Some(PrivateAPIAccess(Seq.empty))
     }
 
     "read from JSON when the API access type is PRIVATE and there is an empty whitelist and isTrial is true" in {
-      val access = """{
-          | "type": "PRIVATE",
-          | "whitelistedApplicationIds": [],
-          | "isTrial": true
-      }""".stripMargin
-      val apiDefinition = anApiDefinition(access = access)
+      val apiDefinition = anApiDefinition(
+        accessType = "PRIVATE", whitelistedApplicationIds = Some("[]"), isTrial = Some(true))
 
       apiDefinition.versions.head.access shouldBe Some(PrivateAPIAccess(Seq.empty, Some(true)))
     }
 
     "read from JSON when the API access type is PRIVATE and there is a non-empty whitelist" in {
-      val access = """{
-          | "type": "PRIVATE",
-          | "whitelistedApplicationIds": ["an-application-id"]
-      }""".stripMargin
-
-      val apiDefinition = anApiDefinition(access = access)
+      val apiDefinition = anApiDefinition(accessType = "PRIVATE", whitelistedApplicationIds = Some("[\"an-application-id\"]"))
 
       apiDefinition.versions.head.access shouldBe Some(PrivateAPIAccess(Seq("an-application-id")))
     }
 
     "fail to read from JSON when the API access type is PRIVATE and there is no whitelist" in {
-      val access = """ { "type": "PRIVATE" } """
-
-      intercept[RuntimeException] { anApiDefinition(access = access) }
+      intercept[RuntimeException] {
+        anApiDefinition(accessType = "PRIVATE", whitelistedApplicationIds = None)
```

Comment:
Why define whitelistedApplicationIds as it defaults to None?

Reply:
Because it makes what it is testing explicit, providing a clear match between test description and code.  Not having it present also means that there is implicit knowledge of the code being called and resilience to the default being changed.  The default is only present for tests that don't care about it.
If IntelliJ complained about it, I would have removed it to satisfy the green tick policy, but I prefer the explicitness.